ui/vite.config.js

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from 'vite'
+import vue from '@vitejs/plugin-vue'
+import { fileURLToPath, URL } from "node:url";
+
+// https://vite.dev/config/
+export default defineConfig({
+    resolve: {
+        alias: {
+            "@": fileURLToPath(new URL("./src", import.meta.url)),
+        },
+    },
+    plugins: [vue()],
+    server: {
+        port: 8080,
+        proxy: {
```

Comment:
I think we have the same problem with sirepo. This proxy should go through the api server. I realize this is a change of scope.

Reply:
I moved this to #82